data/Templates/FhirToHL7v2/ADT_A08.liquid

```diff
@@ -1,5 +1,9 @@
 {% comment %} {{ '' | debug }} {% endcomment %}
 {%- assign messageHeaderObject = fhirData.entry | select: 'resource.resourceType', 'MessageHeader' | get_index: 0 -%}
+{% comment %} Since this is the ADT_A08 liquid template, the following 3 can be hardcoded {% endcomment %}
```

Comment:
I _think_ this comment might be the culprit of your extra newline.

Reply:
Shoot. It is, should have tested after adding the comment. I'll just get rid of it.

Reply:
Your other option is to add the dashes, similar to some of the other ones:

{%- -%}

Reply:
I agree @ accidentally

Reply:
thanks guys, that worked!